src/components/NavBar/NavBar.module.css

```diff
@@ -0,0 +1,35 @@
+.navBar {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 16px;
+    background-color: #d4d4d4;
+    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+    width: 100%;
+    border-radius: 10px;
+    margin-right: 10px;
+    height: 70px;
+    box-sizing: border-box;
+  }
```

Comment:
In my opinion, it is better to use the height rather than padding in order to get a size for the navbar.
So, if you would use ```padding: 0px 16px```, then you can add a little bit of height instead of the padding, and also the box-sizing is almost un-noticable and not needed even. Think about it.
Moreover, is the margin-right really necessary for anything?

Reply:
margin right at this point is a little bit important. since the logo has a left margin of a 10px, this sets the overall elements in the center.